src/containers/walletContainer.js

```diff
@@ -75,6 +73,7 @@ class WalletContainer extends Component {
 
 
     renderEditable(cellInfo, flag) {
+        console.log(this.props)
```

Comment:
please remove all console.logs

Reply:
console.log is removed already at another commit